src/PageContent/AssignVolunteers/AssignOption.js

```diff
@@ -44,11 +44,89 @@ class AssignOption extends Component {
                         
                     </div>
+                    
+                    <div className="avi-detail avi-volunteers">
+                        {deliverers && deliverers.length === 2 && 
+                            <button type="button" 
+                                className="form-button copy" 
+                                id={this.props.deliveryId} 
+                                onClick={this.copyLinkToClipboard.bind(this)}
+                                onMouseEnter={this.recopyDialog.bind(this)}>Copy link</button>
+                        }
+                    </div>
                 </div>
             </div>
         );
     }
 
+
+
+    copyLinkToClipboard(e,f) {
+        // stolen from https://stackoverflow.com/a/30810322
+        var textArea = document.createElement('textarea');
+
+        //
+        // *** This styling is an extra step which is likely not required. ***
+        //
+        // Why is it here? To ensure:
+        // 1. the element is able to have focus and selection.
+        // 2. if element was to flash render it has minimal visual impact.
+        // 3. less flakyness with selection and copying which **might** occur if
+        //    the textarea element is not visible.
+        //
+        // The likelihood is the element won't even render, not even a flash,
+        // so some of these are just precautions. However in IE the element
+        // is visible whilst the popup box asking the user for permission for
+        // the web page to copy to the clipboard.
+        //
+
+        // Place in top-left corner of screen regardless of scroll position.
+        textArea.style.position = 'fixed';
+        textArea.style.top = 0;
+        textArea.style.left = 0;
+
+        // Ensure it has a small width and height. Setting to 1px / 1em
+        // doesn't work as this gives a negative w/h on some browsers.
+        textArea.style.width = '2em';
+        textArea.style.height = '2em';
+
+        // We don't need padding, reducing the size if it does flash render.
+        textArea.style.padding = 0;
+
+        // Clean up any borders.
+        textArea.style.border = 'none';
+        textArea.style.outline = 'none';
+        textArea.style.boxShadow = 'none';
+
+        // Avoid flash of white box if rendered for any reason.
+        textArea.style.background = 'transparent';
+
+
+        textArea.value = `${window.location.origin}/mobile/delivery/${this.props.deliveryId}`;
+
+        document.body.appendChild(textArea);
+        textArea.focus();
+        textArea.select();
+
+        try {
+            var successful = document.execCommand('copy');
+        } catch (err) {
+            return;
+        }
+
+        document.body.removeChild(textArea);
+
+        // update button text to show that it's been copied
+        e.target.className = 'form-button copy copied'; 
+        e.target.innerHTML = 'Link copied to clipboard';
```

Comment:
Maybe instead of "Link copied to clipboard" change it to "It's copied!"
The button is buggy when the text changes because of its height so shortening it could be a solution, but I also wonder if its clear for the users that it is in their clipboard.